src/assets/sass/protocol/components/_breadcrumb.scss

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+.mzp-c-breadcrumb {
+    background-color: $color-light-gray-20;
+    margin-bottom: 0;
+    padding: $spacing-sm $spacing-md;
+
+    .mzp-c-breadcrumb-item {
+        display: inline-block;
+
+        &:first-child {
+            font-weight: 400;
+        }
+
+        + .mzp-c-breadcrumb-item {
+            font-weight: bold;
+
+            &:before {
+                @include bidi(((content, "\003E", "\003C"),));
+                font-weight: normal;
+                margin: 0 0.25em;
```

Comment:
Nit: omit leading zeroes on decimals (doesn't matter in the grand scheme but we should be consistent).